pkg/kubectl/cmd/portforward.go

```diff
@@ -17,21 +17,33 @@ limitations under the License.
 package cmd
 
 import (
+	"fmt"
 	"io"
 	"net/url"
 	"os"
 	"os/signal"
 
-	"github.com/golang/glog"
 	"github.com/renstrom/dedent"
 	"github.com/spf13/cobra"
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/client/restclient"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
 	"k8s.io/kubernetes/pkg/client/unversioned/portforward"
 	"k8s.io/kubernetes/pkg/client/unversioned/remotecommand"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 )
 
+type PortForwardOptions struct {
```

Comment:
By the specification ```<CommandName>Config``` struct (here ```PortForwardOption```) should contain all variables required for the command to run (to ease testing). Building this I was also inspired by the ```LogsOptions``` struct https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/logs.go#L56 used as example.